PowerShell/ScubaGear/Testing/Unit/PowerShell/Providers/AADProvider/Invoke-GraphDirectly.ps1

```diff
@@ -0,0 +1,22 @@
+$ProviderPath = '../../../../../Modules/Providers'
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "$($ProviderPath)/ExportAADProvider.psm1") -Function 'Get-PrivilegedRole' -Force
+
+InModuleScope ExportAADProvider {
+    Describe -Tag 'AADProvider' -Name "Invoke-GraphDirectly" {
+        BeforeAll {
+            #function Invoke-MgGraphRequest {return @{Value = @{cowsound = "moo"}}
```

Comment:
<3

Reply:
Should I change it to a different barnyard denizen?